.buildkite/pipeline.full.yml

```diff
@@ -37,24 +36,23 @@ steps:
   # XcFramework e2e tests
   #
 
-  - label: ':bitbar: {{matrix}} XcFramework barebone e2e tests'
+  - label: ":bitbar: {{matrix}} XcFramework barebone e2e tests"
     matrix:
       - "IOS_17"
```

Comment:
I'm guessing we missed this when iOS 18 was added and it should now be bumped to 26? (So we test earliest and latest versions).